<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QMainWindow" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>543</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1055</width>
    <height>543</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1055</width>
    <height>543</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>frmMain</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgMail.qrc">
    <normaloff>:/OIPfonfoOk.png</normaloff>:/OIPfonfoOk.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblWelcome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1001</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(92, 155, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Welcome to iMessage Simulator!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblInbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>1001</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pshBtnOpen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;Bahnschrift&quot;;
background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pshBtnDelete">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;Bahnschrift&quot;;
background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pshBtnConnect">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;Bahnschrift&quot;;
background-color: rgb(5, 168, 255);</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
font: 75 22pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="text">
     <string>Inbox</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImg">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imgMail.qrc">:/OIPfonfoOk.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuWindows">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionServer"/>
    <addaction name="actionSent"/>
   </widget>
   <widget class="QMenu" name="mnuAbout_us">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Deleted</string>
    </property>
    <addaction name="actionDeleted_Mails"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionSave_Mails"/>
   </widget>
   <widget class="QMenu" name="menuAbout_us">
    <property name="title">
     <string>About us</string>
    </property>
    <addaction name="actionInfo_about_the_app"/>
   </widget>
   <addaction name="mnuWindows"/>
   <addaction name="mnuAbout_us"/>
   <addaction name="menuSave"/>
   <addaction name="menuAbout_us"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionServer">
   <property name="text">
    <string>Server</string>
   </property>
  </action>
  <action name="actionSent">
   <property name="text">
    <string>Sent</string>
   </property>
  </action>
  <action name="actionServers">
   <property name="text">
    <string>Servers</string>
   </property>
  </action>
  <action name="actionUsers">
   <property name="text">
    <string>Users</string>
   </property>
  </action>
  <action name="actionSave_Mails">
   <property name="text">
    <string>Save Mails</string>
   </property>
  </action>
  <action name="actionDeleted_Mails">
   <property name="text">
    <string>Deleted Mails</string>
   </property>
  </action>
  <action name="actionInfo_about_the_app">
   <property name="text">
    <string>Info about the app</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgMail.qrc"/>
 </resources>
 <connections/>
</ui>
